Fix misleading information regarding default value of `lang` attribute (#32604)
mdn/content: files/en-us/web/html/global_attributes/lang/index.md

The **`lang`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) helps define the language of an element: the language that non-editable elements are written in, or the language that the editable elements should be written in by the user. The attribute contains a single "language tag" in the format defined in {{RFC(5646, "Tags for Identifying Languages (also known as BCP 47)")}}.

> > **Note:** The default value of `lang` is the empty string. It is recommended to always specify this attribute with the appropriate value.

Author: ```suggestion
> **Note:** The default value of `lang` is the empty string which means the language is unknown. It is recommended to always specify this attribute with the appropriate value.
```

> > **Note:** The default value of `lang` is the empty string which means the language is unknown. It is recommended to always specify this attribute with the appropriate value.

Author: ```suggestion
> **Note:** The default value of `lang` is the empty string which means the language is unknown, therefore it is recommended to always specify this attribute with the appropriate value.
```

> > **Note:** The default value of `lang` is the empty string which means the language is unknown, therefore it is recommended to always specify this attribute with the appropriate value.

Reviewer: @silverwind what do you think about this with a slight tweak:
```suggestion
> **Note:** The default value of `lang` is the empty string, which means that the language is unknown. Therefore, it is recommended to always specify an appropriate value for this attribute.
```

Author: Sounds better, thanks.
